### PR TITLE
docs(model-3d-viewer): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-545.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-545.gearboxjs.preview.cogniteapp.com

File: src/components/Model3DViewer/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  </Story>
+</Preview>
+
+## Available props
+
+<Props of={Model3DViewer} />
+
+##### `assetId` prop explanation
+
+This prop is needed to control highlighting of asset node in the model right after init process or by changing `assetId` prop after viewer has been initialized.
```
Comment (polomani) [inline src/components/Model3DViewer/stories/basic.story.mdx]: please move this inside `<Props />` I don't see a point of having it here separately. Just replace current text with this:
```
this is used to control highlighting of asset node in the model right after init process or by changing `assetId` prop after viewer has been initialized
```

Comment (polomani) [inline src/components/Model3DViewer/stories/basic.story.mdx]: please move this inside `<Props />` I don't see a point of having it here separately. Just replace current text with this:
```
this is used to control highlighting of asset node in the model right after init process or by changing `assetId` prop after viewer has been initialized
```

Comment (polomani) [reply src/components/Model3DViewer/stories/basic.story.mdx]: I know it's what you've done, it's just good thing to fix

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-545.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
screenshot still does not work properly. Image remains visible after changing component story.

**Comment by amevck** [comment]
> screenshot still does not work properly. Image remains visible after changing component story.

In the original story code and this code call-back function is

```typescript
(url: string) => {
   const img = document.createElement('img');
   img.src = url;
   document.body.append(img);
}
```

There is no way to remove the img from dom (now). It is good if we can change this logic to download image after captured. We can do this on #553

**Comment by mrtraser** [comment]
> There is no way to remove the img from dom (now)
 
You definitely can remove the image from DOM ) But I agree that we can do this on #553

**Pull request merged**